bin/lamassu-revoke

```diff
@@ -12,9 +14,50 @@ if (argv.length !== 1) {
 
 const name = argv[0]
 
-const sql = `delete from user_tokens where name = $1 RETURNING *;`
+const userTokenSql = `SELECT * FROM user_tokens WHERE name = $1`
 
-return db.result(sql, [name], r => r.rowCount)
-    .then(r => {if(r==0){console.log('could not find user.')}
-            else{console.log(name + ' removed successfully.')}})
-.then(() => process.exit(0))
+return db.result(userTokenSql, [name])
+  .then(res => {
+    if (res.rowCount === 0) {
+      console.log(`User ${name} was not found!`)
+      process.exit(0)
+    }
+
+    return _.map(t => t.token, res.rows)
+  })
+  .then(tokens => {
+    const tokenClause = _.map(pgp.as.text, tokens).join(',')
+
+    return db.tx(t => {
+      const smsOverride = t.any(`UPDATE customers SET sms_override_by = null WHERE sms_override_by IN ($1^) RETURNING *`, tokenClause)
+      const idDataOverride = t.any(`UPDATE customers SET id_card_data_override_by = null WHERE id_card_data_override_by IN ($1^) RETURNING *`, tokenClause)
+      const idPhotoOverride = t.any(`UPDATE customers SET id_card_photo_override_by = null WHERE id_card_photo_override_by IN ($1^) RETURNING *`, tokenClause)
+      const cameraOverride = t.any(`UPDATE customers SET front_camera_override_by = null WHERE front_camera_override_by IN ($1^) RETURNING *`, tokenClause)
+      const sanctionsOverride = t.any(`UPDATE customers SET sanctions_override_by = null WHERE sanctions_override_by IN ($1^) RETURNING *`, tokenClause)
+      const authorizedOverride = t.any(`UPDATE customers SET authorized_override_by = null WHERE authorized_override_by IN ($1^) RETURNING *`, tokenClause)
+      const usSsnOverride = t.any(`UPDATE customers SET us_ssn_override_by = null WHERE us_ssn_override_by IN ($1^)`, tokenClause)
+      const tokenDeletion = t.result(`DELETE FROM user_tokens WHERE name = $1 RETURNING *`, [name], r => r.rowCount)
+
+      return t.batch([
```

Comment:
Can't this try to delete the token before cleaning up the customer table? I don't think order is guaranteed here. Probably better to have token deletion part on a inner `then`, keep in mind that line 39 already executes the query, so even that has to be inside the `then`.